readme.txt

```diff
@@ -9,16 +11,60 @@ Smaily newsletter subscription plugin for WordPress
 
 == Description ==
 
-Smaily newsletter subscription plugin for WordPress.
+Smaily email marketing and automation extension plugin for Wordpress.
+Automatically generate a simple sign-up form and add new subscribers to Smaily subscribers list.
+
+= Features =
+
+**Wordpress Newsletter Subscribers**
+
+- Add new subscribers to Smaily subscribers list.
+- Select autoresponder to send automated emails.
+
+**Subscription Widget**
+
+- Option for custom advanced HTML form.
+- Functional even with NoScript users.
+- Supports sending custom form values to Smaily.
+
+= Requirements =
+
+- PHP 5.6+
+- Wordpress 4.0+
+
+= Documentation & Support =
+
+Online documentation with help is available at the [Knowledgebase](http://help.smaily.com/en/support/home).
+
+= Contribute =
+
+All development for Smaily for Wordpress is handled via [GitHub](https://github.com/sendsmaily/sendsmaily-wordpress-plugin/).
+Opening new issues and submitting pull requests are welcome.
 
 == Installation ==
 
-* Install as regular plugin.
-* Open smaily plugin settings from admin side panel menu "Smaily".
-* Validate your credentials.
-* Go to Appearance -> Widgets.
-* Place "Smaily newsletter subscription" widget where you wish.
-* Using advanced tab in smaily settings will render advanced form.
+1. Upload or extract the `sendsmaily-wordpress-plugin` folder to your site's `/wp-content/plugins/` directory. You can also use the *Add new- option found in the *Plugins- menu in WordPress.
+2. Activate the plugin from the *Plugins- menu in WordPress.
+
+== Frequently Asked Questions ==
+
+= How to set up widget for signup form? =
+
+1. Validate your smaily credentials in Smaily settings menu.
+2. Move to Appearance -> Widget menu from admin page sidepanel.
+3. Add Smaily Newsletter widget to your prefered location on page.
+4. Select Title for your subscribe newsletter form.
+
+If you have added Form Submitted automation trigger from Smaily site under Automation tab you can see all availabe autoresponders in your widget settings.
+
+There is no need to select autoresponder for widget form, but if you want to customize different approach from opt-in automation trigger you can do that.
+When no autoresponder selected regular opt-in workflow will run. You can add delay, filter by field and send email after subscription. For that edit settings in Smaily automation page.
+
+== Screenshots ==
+
+1. Smaily plugin in admin view.
```

Comment:
It would be great if you could add screenshot of admin view after credentials have been validated.